docs/source/developer/mission.rst

```diff
@@ -0,0 +1,76 @@
+Numba Mission Statement
+=======================
+
+Introduction
+------------
+
+This document is the mission statement for the Numba project. It exists to
+provide a clear description of the purposes and goals of the project. This
+document begins by providing some background on use-cases for Numba and
+categories of users. With this context established it continues with a
+description of the overall goals of the Numba project and its future direction.
+
+This is a living document:
+
+=========================== ============
+The first revision date is: March 2022
+The last updated date is:   March 2022
+The next review date is:    TBC
+=========================== ============
+
+Background
+----------
+
+The Numba project provides tools to improve the performance of scientific
+software. It comprises numerous facilities including just-in-time (JIT)
+compilation, extension points for scientific library writers, and a compiler
+toolkit on which new computational acceleration technologies can be explored
+and built.
+
+The user base of Numba includes anyone needing to perform intensive
+computational work, including scientists and practitioners from a wide range of
```

Comment:
When I see "scientific" repeated throughout this document and use base as "including scientists" as the first example, that implies to me the project has a research first focus. I would like to reduce some of the usage of the word "scientific" to make this appear perhaps more balanced between scientists and practitioners.

For example, I don't think the scientific is essential in the examples and I think "scientific library writers" could be more generically "library writers".